test/cpp/microbenchmarks/bm_fullstack_trickle.cc

```diff
@@ -203,10 +211,11 @@ class TrickledCHTTP2 : public EndpointPairFixture {
   std::unique_ptr<std::ofstream> log_;
   gpr_timespec start_ = gpr_now(GPR_CLOCK_MONOTONIC);
 
-  grpc_endpoint_pair MakeEndpoints(size_t kilobits) {
+  static grpc_endpoint_pair MakeEndpoints(size_t kilobits,
```

Comment:
The static was a very nice touch to explicitly identify that this won't touch any member variables